src/sockaddr.rs

```diff
@@ -14,6 +14,7 @@ use crate::sys::{
 ///
 /// `SockAddr`s may be constructed directly to and from the standard library
 /// [`SocketAddr`], [`SocketAddrV4`], and [`SocketAddrV6`] types.
+#[derive(clone::Clone)]
```

Comment:
`Clone` is part of the prelude so we don't the import `clone`.

```suggestion
#[derive(Clone)]
```